.github/workflows/dep_rust.yml

```diff
@@ -97,9 +97,17 @@ jobs:
       - name: Build Rust Wasm examples
         run:  just build-rust-wasm-examples ${{ matrix.config }}
         working-directory: ./src/hyperlight_wasm
+      
+      - name: Show disk usage
+        run: |
+          df -h
+          du --max-depth=3 -h $HOME | sort -h
+          du --max-depth=3 -h ./target | sort -h
 
       - name: Test
-        run:  just test ${{ matrix.config }}  ${{ matrix.hypervisor == 'mshv3' && 'mshv3' || ''}}
+        run:  |
+          cargo clean -p hyperlight-wasm
```

Comment:
do we want to keep these with the update to increase the image drive size?

Reply:
Probably not, it's potentially making build times longer.